src/backend/app/Http/Middleware/UserDefaultDatabaseConnectionMiddleware.php

```diff
@@ -30,6 +30,10 @@ public function handle($request, \Closure $next) {
     }
 
     private function handleApiRequest(Request $request, \Closure $next) {
+        // health check route
+        if ($request->path() === 'up') {
+            return $next($request);
+        }
```

Comment:
Could you move this down to around line 62 where we are defining other "custom" Laravel routes like `horizon` and `laravel-erd` ?